extra_tests/snippets/stdlib_datetime.py

```diff
@@ -131,6 +131,11 @@ def __init__(self, offset, name):
 assert_raises(NotImplementedError, ne.utcoffset, dt)
 assert_raises(NotImplementedError, ne.dst, dt)
 
+# unsupport format in strptime returns arg itself
+# in linux. Todo: add cases for Mac/Windows
+if sys.platform == 'linux':
```

Comment:
```suggestion
if sys.platform.startswith("linux"):
```
I found `linux` is not the exact name but prefix